cmake/README.md

```diff
@@ -0,0 +1,38 @@
+## CMake for Mbed OS
+
+Until further notice, CMake for Mbed OS is still in the development. 
+
+### Supported targets
+
+Only a limited set of targets is supported at the moment.
+
+These targets should work:
+- K64F 
+- DISCO_L475VG_IOT01A 
+- NRF52840_DK 
+- WIO_3G
+- K66F
+
+### Supported toolchains
+
+Arm Compiler 6 and GNU Arm Embedded toolchains are supported.
+
+### Examples
+
+Supported examples can be identified by the presence of a top level `CMakelists.txt` input source file. 
+
+### Known issues
+
+- ARMClang6: the assembler does not include definitions. It will be fixed in the upcoming release (>3.18.2). Reference: https://gitlab.kitware.com/cmake/cmake/-/issues/21148#note_823305
+- Ninja: ARMClang6 response files require unix paths on Windows. Reference: https://gitlab.kitware.com/cmake/cmake/-/issues/21093
+
+## How to build an application
+
+Prerequisities:
+- CMake >=3.18.2
+- mbed-tools >=3.2.0
+
+From the application root run the following:
+1. To create Mbed OS configuration: `mbedtools configure -m target_name -t toolchain_name`
```

Comment:
```suggestion
1. To create the Mbed OS configuration CMake module, from the root of the application (or wherever `mbed-os.lib` is found), run: 
``
mbedtools configure -m <mbed-target> -t <toolchain>
``
```
Please note that I omitted the extras "`" for formatting purposes in this comment.